### ISSUE TITLE
[scs] SCs builder wrong behaviour if sentence starts from file

### ISSUE THREAD

**Issue by ShunkevichDV** [ostis-ai/sc-machine#39]
**Describe the bug**
Builder incorrectly processes scs-sentence if it starts from file (both kind of file representation, see examples)

**To Reproduce**
Try these sentences^

[IMS.ostis]
<- file;
=> nrel_sokraschenie:
    [Метасистема IMS.ostis];;

"file://files/file0.html"
<- file;
=> nrel_sokraschenie:
    "file://files/file1.html";;

Builder not fails, but output construction in memory is quite strange in both cases. 

**Expected behavior**
It is expected, that builder creates two files in sc-memory with correspondent content and connects them with relation.

**Screenshots**
First case (sc-element with system identifier [IMS.ostis] was created)
![изображение](https://user-images.githubusercontent.com/1109638/163596147-a7e5e137-41df-4f3c-bf8b-792d0ef60bb6.png)

Second case (sc-element with system identifier "file://files/file0.html" was created)
![изображение](https://user-images.githubusercontent.com/1109638/163596350-e95292ea-0451-4b41-9440-b3e6196398e2.png)

**Comment by NikitaZotov** [comment]
I tried to reproduce the problem, but nothing worked. I created a test fragment and found it in the editor. Here's what happened:
![image](https://user-images.githubusercontent.com/56268020/170204587-eaa7e340-a1e2-4049-9708-102385fa409b.png)
![image](https://user-images.githubusercontent.com/56268020/170204646-3667c403-bde4-48d3-9c5e-fc6ecebb6dde.png)
![image](https://user-images.githubusercontent.com/56268020/170204714-9ae4a5b0-4bea-4c40-865a-ca87ccb9db50.png)
Are you sure that this problem hasn't any hidden context?
